### PR TITLE
[stable/openebs] update charts to 1.1.0 release

### PR DESCRIPTION
**Pull request by ChandanSagar**
- Updated README
- Updated chart version to 1.1.0
- Updated values.yaml

Signed-off-by: ChandanSagar <chandan.pradhan@mayadata.io>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Review by AmitKumarDas — APPROVED** [review]

File: stable/openebs/README.md [hunk 1]
```diff
@@ -40,49 +40,52 @@ The following table lists the configurable parameters of the OpenEBS chart and t
...
-| `cstor.volumeMgmt.imageTag`             | Image Tag for cStor Volume Management         | `1.0.0`                                   |
+| `cstor.volumeMgmt.imageTag`             | Image Tag for cStor Volume Management         | `1.1.0`                                   |
 | `policies.monitoring.image`             | Image for Prometheus Exporter                 | `quay.io/openebs/m-exporter`              |
-| `policies.monitoring.imageTag`          | Image Tag for Prometheus Exporter             | `1.0.0`                                   |
+| `policies.monitoring.imageTag`          | Image Tag for Prometheus Exporter             | `1.1.0`                                   |
 | `analytics.enabled`                     | Enable sending stats to Google Analytics      | `true`                                    |
 | `analytics.pingInterval`                | Duration(hours) between sending ping stat     | `24h`                                     |
+| `defaultStorageConfig.enabled`          | Enable default storage class installation     | `false`                                   |
```
Comment (kmova) [inline stable/openebs/README.md]: The default value should be `true`

File: stable/openebs/values.yaml [hunk 2]
```diff
@@ -33,9 +33,12 @@ apiserver:
     initialDelaySeconds: 30
     periodSeconds: 60
 
+defaultStorageConfig:
+  enabled: "false"
```
Comment (kmova) [inline stable/openebs/values.yaml]: default should be `true` to keep it backward compatible.

**Review by prateekpandey14 — APPROVED** [review]

**Review by kmova — APPROVED** [review]
/lgtm
/approved

**Pull request merged**